migrations/20231027141322_add_session_refresh_columns.up.sql

```diff
@@ -0,0 +1,4 @@
+alter table if exists {{ index .Options "Namespace" }}.sessions
+  add column if not exists refreshed_at timestamp without time zone,
+  add column if not exists user_agent text,
+  add column if not exists ip inet;
```

Comment:
it doesn't seem like the inactivity timeout feature uses the user_agent or the ip - is it necessary to include this data in this PR? to err on the side of caution of storing redundant information in the db, i would prefer to only include these db fields in a separate PR

Reply:
The reason why it's added here is that gathering this information should start sooner or later. Whether it's in this PR or another only has value for us as developers, and I don't think we gain anything major by moving it to another PR at this point.